### DISCUSSION TITLE
How to build Flet on pc?

### DISCUSSION THREAD

**Discussion by Wanna-Pizza** [flet-dev/flet#4216] [Q&A]
### Question

**How to build Flet on pc. and then use it like library?** 

Instruction very unclear to me. 

**https://github.com/flet-dev/flet/blob/main/CONTRIBUTING.md**

I want to build **flet** at my pc, without uploading to github, and so on, bcoz **I JUST DONT UNDERSTAND.** 

And there is no video about how to do this. I really dont understand anything. 

Just how to pack, compile flet ? 

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by ndonkoHenri** [comment]
That file you linked is to be used only when contributing to the Flet framework itself. (I guess that's not your case)

To run a Flet app on your computer follow  this: https://flet.dev/docs/getting-started/

To package a Flet app for a particular platform: https://flet.dev/docs/publish

Feel free to ask further questions, while being specific on what you are trying to achieve.

**Reply by Wanna-Pizza** [reply]
> That file you linked is to be used only when contributing to the Flet framework itself. (I guess that's not your case)
> 
> To run a Flet app on your computer follow this: https://flet.dev/docs/getting-started/
> 
> To package a Flet app for a particular platform: https://flet.dev/docs/publish
> 
> Feel free to ask further questions, while being specific on what you are trying to achieve.

Thank you! But i dont think i need this one. 

Im trying to get full library Flet, to run my own projects without packing it. 

Just like when I use `pip install flet`

But I added new packages with dart, and I want to other ppl use it. So I guess I need compile flutter/dart into flet LIB

**Reply by Wanna-Pizza** [reply]
> That file you linked is to be used only when contributing to the Flet framework itself. (I guess that's not your case)
> 
> To run a Flet app on your computer follow this: https://flet.dev/docs/getting-started/
> 
> To package a Flet app for a particular platform: https://flet.dev/docs/publish
> 
> Feel free to ask further questions, while being specific on what you are trying to achieve.



So when you do `pip install flet` it install full library that you can use. 

I want to get full library from new packages that I add with dart. Like full flet client. To use on others pc just with python.

**Reply by Wanna-Pizza** [reply]
> That file you linked is to be used only when contributing to the Flet framework itself. (I guess that's not your case)
> 
> To run a Flet app on your computer follow this: https://flet.dev/docs/getting-started/
> 
> To package a Flet app for a particular platform: https://flet.dev/docs/publish
> 
> Feel free to ask further questions, while being specific on what you are trying to achieve.

I want to save modified Flet as library. Without running flutter servers, just like same when we do Pip install flet

**Reply by ndonkoHenri** [reply]
https://flet.dev/docs/extend/integrating-existing-flutter-packages ?
